src/demo/styles.css

```diff
@@ -41,16 +41,6 @@ section {
   max-width: 100%;
   height: 100%;
 }
-.demo prx-datepicker {
```

Comment:
these are old leftovers from when I first started making demos in the styleguide